### PR TITLE
Affine cipher exercise

### PR DESCRIPTION
**Pull request by tofische**
Added a new Haskell practice exercise Affine Cipher based on the available problem specification.

### CODE REVIEW

File: exercises/practice/affine-cipher/.meta/config.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "authors": [],
```
Comment (BNAndras) [inline exercises/practice/affine-cipher/.meta/config.json:2]: Not a Haskell maintainer here, but this should contain your GitHub username.

Comment (tofische) [reply exercises/practice/affine-cipher/.meta/config.json:2]: You're right, corrected.

**Comment by MatthijsBlom** [comment]
It looks like you included `rational-numbers` by accident.

**Comment by tofische** [comment]
> It looks like you included `rational-numbers` by accident.

As I explained [here](https://forum.exercism.org/t/affine-cipher-exercise/9320/3), it was my mistake because I created the affine-cipher-exercise branch from the not-yet-merged rational-numbers-exercise branch and not from the main branch.

If I understand git properly, this PR can be merged without problems after Rational Numbers is merged, so no corrective action is necessary - is this assumption correct?

**Comment by MatthijsBlom** [comment]
I suspect so. [`git rebase --onto`](https://git-scm.com/docs/git-rebase) should also work, I think.

**Comment by tofische** [comment]
> I suspect so. [`git rebase --onto`](https://git-scm.com/docs/git-rebase) should also work, I think.

It shall be corrected now.

**Comment by tofische** [comment]
PR is obsolete.

**Pull request closed without merging**